Add caller error store to error handling (#15)
aeternity/mnesia_rocksdb: README.md

 | error   | always               | always                 | exception |
 | fatal   | always               | always                 | core dump |

### Option 2: `on_write_error_store`

An `on_write_error_store` option can be provided, per-table, in the `rocksdb_opts`
user property (see [Customization](#customization) above).
When set, the backend will use the value of the option as the name for an ETS table
which is used as storage for runtime write errors. The table must be set up outside
of the backend by the clients themselves.

Entries to the table are in the form of a tuple `{{Table, Key}, Error, InsertedAt}`
where `Table` refers to the Mnesia table name, `Key` is the primary key being used by Mnesia,

> `Error` is the error the backend encounteredA and `InsertedAt` refers to the time

Reviewer: ```suggestion
`Error` is the error encountered by the backend, and `InsertedAt` refers to the time
```

> the error was encounterd as system time in milliseconds.

Reviewer: ```suggestion
the error was encountered as system time in milliseconds.
```

> The backend will only insert entries and otherwise not managed the table. Thus, clients

Reviewer 2: ```suggestion
The backend will only insert entries and otherwise not manage the table. Thus, clients
```
